src/Servers/HttpSys/src/HttpSysOptions.cs

```diff
@@ -229,6 +229,11 @@ public Http503VerbosityLevel Http503Verbosity
         /// </remarks>
         public bool UnsafePreferInlineScheduling { get; set; }
 
+        /// <summary>
+        /// Configures request headers to use Latin1 encoding.
```

Comment:
Defaults to false, and utf-8 will be used instead.

Reply:
https://github.com/dotnet/aspnetcore/pull/34568/commits/bd8d56096cc5c8a4a0049aa88579f03400d65323